lib/media/track/localvideotrack.js

```diff
@@ -92,6 +111,9 @@ class LocalVideoTrack extends LocalMediaVideoTrack {
    * @fires VideoTrack#enabled
    */
   enable() {
```

Comment:
I think this can be reworded to:
```
Enable the {@link LocalVideoTrack}. This is effectively "unpause". If a {@link VideoProcessor} is added, then
`processedTrack` is enabled as well. 
```